package/boot/uboot-rockchip/Makefile

```diff
@@ -5,10 +5,10 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
-PKG_VERSION:=2020.07
-PKG_RELEASE:=3
```

Comment:
This needs the updated generated platform-data for the SPL, which is missing in this PR.